package.json

```diff
@@ -3,14 +3,17 @@
   "scripts": {
     "test": "node playwright-single.js",
     "test-smartui": "node playwright-smartui.js",
-    "test-parallel": "node playwright-parallel.js"
+    "test-parallel": "node playwright-parallel.js",
+    "test-android": "node playwright-android-real-device.js",
+    "test-ios": "node playwright-single-on-iphone.js",
```

Comment:
Can you remove the ios test command for now? As this is a playwright test with emulation, not a real device with ios.